Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8/nodejs-12:1-98.1634036321 as builder
+
+WORKDIR /opt/app-root/src
+
+RUN npm install --global yarn
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install
+COPY . .
+RUN yarn build
+
+CMD yarn serve
```

Comment:
Actually, I know there are many discussions between both tools but I don't know the differences.

The console uses yarn so I wanted to stay with the same tools.

Reply:
Yes, yarn is supposed to be faster :1st_place_medal: but they actually have the same features

If you want to use yarn here, I guess we can get a docker image that already have it ?
And maybe avoid usage of npm / npx elsewhere like `"serve": "npx http-server`

Reply:
I did not find any ubi8 image with yarn, this is why I was using this image and installing yarn.
I think we have a choice here (and I do not have opinion about it):
- staying with ubi8 image and installing yarn manually 
- staying with ubi8 and dropping yarn at least in the dockerfile
- moving to another image

I will remove the npx http-serve and switch back to yarn since the dockerfile usr http-serve.sh anyway now.

Reply:
+1 to stay with ubi
for yarn vs npm I don't have a strong opinion